lib/lotus/utils/string.rb

```diff
@@ -69,6 +69,33 @@ def initialize(string)
         @string = string.to_s
       end
 
+      # Detect if a string is nil or empty
+      #
+      # @return [TrueClass, FalseClass] result checked string
+      #
+      # @since x.x.x
+      #
+      # @example nil
+      #   require 'lotus/utils/string'
+      #
+      #   string = Lotus::Utils::String.new nil
+      #   string.blank? # => true
+      #
+      # @example empty string
+      #   require 'lotus/utils/string'
+      #
+      #   string = Lotus::Utils::String.new ''
+      #   string.blank? # => true
+      #
+      # @example non empty string
+      #   require 'lotus/utils/string'
+      #
+      #   string = Lotus::Utils::String.new 'lotus'
+      #   string.blank? # => false
+      def blank?
+        @string.nil? || @string.empty?
```

Comment:
the `nil` scenario is not possible because string is always casted to_s in constructor